docs/nydusd.md

```diff
@@ -265,7 +265,10 @@ Currently, the mirror mode is only tested in the registry backend, and in theory
               // When Dragonfly does not cache data, nydusd will pull it from "X-Dragonfly-Registry".
               // If not set "X-Dragonfly-Registry", Dragonfly will pull data from proxy.registryMirror.url.
               "X-Dragonfly-Registry": "https://index.docker.io"
-            }
+            },
+            // Endpoint of mirror server health checking (including scheme).
```

Comment:
This URL endpoint is used to check the health of mirror server, and if the mirror is unhealthy, the request will 
 fallback to the next mirror or the original registry server. Use `$host/v2` as default if left empty.

Reply:
Already replaced it. Thanks.